pkg/envoy/rds/response.go

```diff
@@ -1,8 +1,10 @@
 package rds
 
 import (
+	mapset "github.com/deckarep/golang-set"
 	xds_discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
+	"github.com/envoyproxy/go-control-plane/pkg/cache/v2"
```

Comment:
Would it make sense to alias this with the name of the package - `cache` or even `cache_v2` or `cache_v3`?

Reply:
misstype from VScode autocompletion